frontend/src/Calendar.js

```diff
@@ -2,19 +2,108 @@ import React from 'react'
 import FullCalendar from '@fullcalendar/react'
 import dayGridPlugin from '@fullcalendar/daygrid'
 
+import moment from 'moment'
+import timeGridPlugin from '@fullcalendar/timegrid'
+
+import {fetchUserCalendar} from './script'
+
+// the conversion of one minute to 60000 milliseconds
+const MIN_TO_MS = 60000;
```

Comment:
No need to change anything, but you can avoid the need to add this as a separate constant/comment by documenting the unit in the variable name (e.g. startTimeMs). You'll also see `meetingTime.start * 60 * 1000` sometimes to more explicitly show the calculation.

Reply:
Thanks for pointing this out.